source/clear-linux/get-started/bootable-usb/bootable-usb.rst

```diff
@@ -176,6 +176,7 @@ Burn the |CL| image onto a USB drive
    |CAUTION-BACKUP-USB|
 
 #. Download the `Rufus`_ utility to burn the image onto a USB drive.
+   **Use only the latest version of Rufus**.
```

Comment:
I would suggest adding a comment of what the latest version is at the time of this publication; otherwise, you're committed to checking for the latest and any possible changes all of the time.

Reply:
I agree here, we should include which version we validated with.

Reply:
I will add the version but retain the note. Users are on notice that they should use the latest Rufus.

Reply:
See latest commit, @evil-brain.